README.md

```diff
@@ -4,7 +4,7 @@
 
 Customizable Icons for Flutter. Port of [react-native-vector-icons](https://github.com/oblador/react-native-vector-icons).
 
-Preview icons with the gallery: https://flutter-vector-icons.vercel.app
+Preview icons with the gallery: [https://flutter-vector-icons.vercel.app](https://flutter-vector-icons.vercel.app/)
```

Comment:
> I replaced the old Preview link with working one (https://flutter-vector-icons.vercel.app/#/)

I can't see any difference between the previous link and the new link.

Reply:
It is not updated in pub.dev

Reply:
That's right. 

I checked and I saw that the latest [release](https://github.com/pd4d10/flutter-vector-icons/releases/tag/v2.0.0) does not contain the latest commits which contain the link update.

So, it looks like the link will update on pub.dev when @pd4d10 creates a new release.